lib/PuppeteerSharp.Tests/Page/EvaluateTests.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class EvaluateTests : PuppeteerBaseTest
+    {
+        [Fact]
+        public async Task ShouldWork()
+        {
+            using (var page = await Browser.NewPageAsync())
+            {
+                var result = await page.EvaluateAsync<int>("() => 7 * 3");
```

Comment:
I'm guessing using `Fact` instead of `Theory` is to stay true to the original? the only difference between the tests in the input of the method `EvaluateAsync` and the expected value

Reply:
Good idea. I'll try to implement that. I'm a Nunit user so I'm trying to get used to XUnit. I'd like to see if it supports types as parameters for the generics.